production/docker/config/loki-docker-memberlist-boltdb-shipper.yaml

```diff
@@ -0,0 +1,108 @@
+auth_enabled: false
+
+http_prefix:
+
+server:
+  http_listen_address: 0.0.0.0
+  grpc_listen_address: 0.0.0.0
+  http_listen_port: 3100
+  grpc_listen_port: 9095
+  log_level: debug
+
+memberlist:
+  join_members: ["loki-1", "loki_2", "loki_3"]
+  dead_node_reclaim_time: 30s
+  gossip_to_dead_nodes_time: 15s
+  left_ingesters_timeout: 30s
+  bind_addr: ['0.0.0.0']
+  bind_port: 7946
+
+ingester:
+  lifecycler:
+    join_after: 60s
+    observe_period: 5s
+    ring:
+      replication_factor: 2
+      kvstore:
+        store: memberlist
+    final_sleep: 0s
+  chunk_idle_period: 30m
+  max_chunk_age: 2h
```

Comment:
for consistency with what we just made the other defaults a few days ago, can you set both of these to `1h`?

Reply:
Fixed